app/models/order.py

```diff
@@ -7,7 +7,7 @@
 class OrderTicket(db.Model):
     __tablename__ = 'orders_tickets'
     order_id = db.Column(db.Integer, db.ForeignKey('orders.id', ondelete='CASCADE'), primary_key=True)
-    ticket_id = db.Column(db.Integer, db.ForeignKey('ticket.id', ondelete='CASCADE'), primary_key=True)
+    ticket_id = db.Column(db.Integer, db.ForeignKey('ticket.id', ondelete='CASCADE'), nullable=True)
```

Comment:
An `ondelete='SET NULL'` would have required a `nullable` column. Whereas `CASCADE` will anyway just delete the record. Which is what we want to happen. Please revert back to the previous state and remove the migrations.